autopilot-daemon/pkg/utils/functions.go

```diff
@@ -87,7 +102,7 @@ func CreateJob(healthcheck string) {
 		klog.Info("Cannot get pod:", err.Error())
 	}
 	autopilotPod := pods.Items[0]
-	ttlsec := int32(4 * 60 * 60) // setting TTL to 4 hours
+	ttlsec := int32(30) // setting TTL to 30 sec
```

Comment:
Is this a mistake? If the `dcgm -r 3` test takes minutes to run, would a 30s TTL terminate the job early?

Reply:
Oh no no this ttl starts when the job is completed. So once dcgm is done and the pod is marked as completed, the pod will stay there for 30s and then will be deleted